pttrack/fixtures/pttrack.json

```diff
@@ -1,4 +1,5 @@
 [
+{% load i18n %}
```

Comment:
Does this work?! I would expect the json parser to choke on this. Does it actually respect the template language?

Reply:
I left this one in the json file by mistake. It's wrong, you are right.